master/networking/vxlan-ipip.md

```diff
@@ -41,6 +41,10 @@ You can configure each IP pool with different encapsulation configurations. Howe
 - [Configure IP in IP encapsulation for all inter workload traffic](#configure-ip-in-ip-encapsulation-for-all-inter-workload-traffic)
 - [Configure VXLAN encapsulation for all inter workload traffic](#configure-vxlan-encapsulation-for-all-inter-workload-traffic)
 
+#### ipv4/6 address support
```

Comment:
I think `ipv4` should be `IPv4`, that seems to be the most common spelling.

Reply:
fixed